measures/measures-schema.yaml

```diff
@@ -287,18 +287,17 @@ definitions:
     anyOf:
       - required: ['procedureCodes']
       - required: ['diagnosisCodes']
-    required: [submissionMethods]
```

Comment:
Why is this not required anymore? Or is what you've added only applicable to claims? I'm forgetting the original intention here - but is it possible that registry measures use different codes?

Reply:
The allowed submission methods are an attribute of measures, not of particular performance or eligibility options. @akegan do you have any thoughts?

Reply:
they are all claims specific, registries do use different codes, but they are not contained in the data at all right now. If we ant to keep submissionMethods as required, we would have to add to performance options too

Reply:
@akegan are you saying that different eligibility options can have different permitted submission methods?